lib/logging.js

```diff
@@ -44,28 +43,28 @@ var makeTransports = function() {
         }));
     }
     if (loggerConfig.logfile) {
-        transports.push(new (winston.transports.File)({
+        transports.push(new (winston.transports.DailyRotateFile)({
             filename: loggerConfig.logfile,
             json: false,
             name: "logfile",
             level: loggerConfig.level,
             timestamp: timestampFn,
             formatter: formatterFn,
-            maxsize: loggerConfig.maxFileSizeBytes,
             maxFiles: loggerConfig.maxFiles,
+            datePattern: ".yyyy-MM-dd",
```

Comment:
The problem with doing this is that winston will adjust its rotation frequency based on this pattern e.g if you give a pattern which includes hours, it'll rotate hourly. I don't really want to preemptively give the ability to configure this because I think it's going to make more hassle as people flood me with complaints about how it isn't rotating daily because the pattern they used doesn't contain `dd` or does contain `HH` or `mm`.